core/server/worker/src/main/java/alluxio/worker/netty/AbstractWriteHandler.java

```diff
@@ -145,9 +145,6 @@ public void channelRead(ChannelHandlerContext ctx, Object object) throws Excepti
         initRequestContext(mContext);
       }
 
-      // Validate the write request.
-      validateWriteRequest(writeRequest, msg.getPayloadDataBuffer());
-
       // If we have seen an error, return early and release the data. This can only
       // happen for those mis-behaving clients who first sends some invalid requests, then
```

Comment:
update the comment here. this is not really about mis-behaving clients but expected case due to async nature.